src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,21 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
 
+@Entity
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+
+    @OneToMany(cascade = CascadeType.MERGE)
```

Comment:
Please re-check the requirements in the task, this entity should not have any CascadeType

Reply:
fixed